blog/2025-03-01-config-flow-unique-id.md

```diff
@@ -0,0 +1,15 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "New checks for config flow unique ID"
+---
+
+### Summary of changes
+
+When a config flow creates an entry with a colliding unique ID, the old entry is currently automatically removed and replaced with the new config entry.
```

Comment:
It's not clear what has changed. We should start with saying what the new behavior is, that required us to publish this blog post.